roles/openshift/tasks/firewalld.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
What is the impetus for combining the openshift-master and openshift-node roles into a single 'openshift' role?

I prefer them separate as they are logically separate roles. One is a master, one is a node.

Now just because they're logically separate roles doesn't mean they can't both be used in a single playbook to setup a single host to do both things (all in one devenv).

Looking at how others are doing it in Ansible Galaxy, they seem to be separating out the roles by sub function like what we're doing before this PR with openshift-master and openshift-node.

For example, Openstack has the following roles on Ansible Galaxy:
openstack-glance
openstack-keystone
openstack-nova_compute

From:
https://galaxy.ansible.com/list#/roles

I couldn't find a way to link to my search, so just search for openstack to see the examples.